core/src/com/nuclearthinking/myheroagency/model/Settings.java

```diff
@@ -0,0 +1,54 @@
+package com.nuclearthinking.myheroagency.model;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Preferences;
+import org.slf4j.Logger;
+import org.slf4j.impl.SimpleLoggerFactory;
+
+import static com.nuclearthinking.myheroagency.utils.Constants.PREFERENCES_NAME;
+
+public class Settings {
+
+    private static final String LANGUAGE_KEY = "language";
+    private static final String DEFAULT_LANGUAGE = "ru";
+    private static Logger logger = new SimpleLoggerFactory().getLogger("Settings");
+
+    private String language;
+
+    private static Settings defaultSettings() {
+        logger.info("Loading default settings");
+        Settings settings = new Settings();
+        settings.setLanguage(DEFAULT_LANGUAGE);
+        return settings;
+    }
+
+    public static Settings loadSettings() {
+        logger.info("Loading settings {} from preferences", PREFERENCES_NAME);
+        Settings settings = new Settings();
+        Preferences preferences = Gdx.app.getPreferences(PREFERENCES_NAME);
+        if (preferences == null) {
+            logger.error("Can't load preferences {}", PREFERENCES_NAME);
+            return defaultSettings();
+        }
+        settings.setLanguage(preferences.getString(LANGUAGE_KEY, DEFAULT_LANGUAGE));
+
+        return settings;
+    }
+
+    public void save() {
```

Comment:
On Android, the system's SharedPreferences class is used. This means preferences will survive app updates, but are deleted when the app is uninstalled.